docs/en/sql-reference/sql-statements/data-manipulation/PAUSE_ROUTINE_LOAD.md

```diff
@@ -15,14 +15,14 @@ This statement pauses a Routine Load job but does not terminate this job. You ca
 ## Syntax
 
 ```SQL
-PAUSE ROUTINE LOAD FOR <db_name>.<job_name>;
+PAUSE ROUTINE LOAD FOR [db_name.]<job_name>;
```

Comment:
the privilege requirement in line 13 is not correct, it's the privilege name in v2.5 and earier